lots.py

```diff
@@ -46,8 +48,12 @@ def cast(self):
         for i in range(grid_side):
             print("".join(self.lots[(i * grid_side):(i * grid_side + grid_side)]))
 
+    def uncast(self):
+        print('\n' + self.neatlots + '\n')
```

Comment:
if I change the number of lots, neatlots won't get updated. gotta calculate this on the fly